data/mods/Magiclysm/monster_factions.json

```diff
@@ -8,6 +8,11 @@
     "name": "dragon_black",
     "friendly": [ "ooze", "lizardfolk" ]
   },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "hunllef",
+    "friendly": [ "herbivore" ]
```

Comment:
```suggestion
    "base_faction": "herbivore"
```

Reply:
You know this section much better than I do. My goal is for them to easily get angry at anything that isn't an herbivore but still be tameable.

Reply:
Hmm... How are other creatures supposed to behave vs them?